### PR TITLE
[NOT AZURE] Azure Stack Update Admin SDK

### PR DESCRIPTION
**Pull request by deathly809**
## Description

This is for Azure Stack **ONLY**.  There seems to be an issue when generating the SDK remotely for the repo some reason.  I have message the Autorest team and ask them for help but no response.

https://github.com/Azure/azure-rest-api-specs/pull/2839

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [X] Please add REST spec PR link to the SDK PR
- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [X] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [X] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [X] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [X] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [X] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by dsgouda** [comment]
@deathly809 please fix compilation errors here

**Comment by deathly809** [comment]
@dsgouda  Fixed!  We fixed an issue with the spec which resulted in a property being removed.

File: src/SDKs/_metadata/azsadmin_resource-manager_update.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+﻿Executing AutoRest command
+cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/azsadmin/resource-manager/update/readme.md --csharp --csharp-sdks-folder=E:\github\azure-sdk-for-net\src\AzureStack\UpdateAdmin\Update.Admin --version=latest --reflect-api-versions
+
```
Comment (dsgouda) [inline src/SDKs/_metadata/azsadmin_resource-manager_update.txt]: Please generate the code using generate.ps1 or generate.cmd and commit those changes.
Looks like autorest failed to generate the code from config file here

**Comment by deathly809** [comment]
@dsgouda Regenerated, hopefully everything works this time.

File: src/SDKs/_metadata/azsadmin_resource-manager_update.txt (hunk 1 above)
Comment (deathly809) [reply src/SDKs/_metadata/azsadmin_resource-manager_update.txt]: Done.

File: src/AzureStack/UpdateAdmin/Update.Admin/Microsoft.AzureStack.Management.Update.Admin.csproj [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    <VersionPrefix>0.1.0-preview</VersionPrefix>
+    <AssemblyName>Microsoft.AzureStack.Management.Update.Admin</AssemblyName>
+    <PackageTags>Microsoft Azure Stack;Update;REST HTTP client;azureostackfficial;netcore451511</PackageTags>
+    <PackageReleaseNotes>This is a preview release of the AzureStack Update Admin SDK.</PackageReleaseNotes>
+  </PropertyGroup>
+  <PropertyGroup>
+    <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Update="node_modules\JSONSelect\site\css\droid_sans.tt">
```
Comment (shahabhijeet) [inline src/AzureStack/UpdateAdmin/Update.Admin/Microsoft.AzureStack.Management.Update.Admin.csproj]: @deathly809 what are these files?

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
It is missing bunch of meta data
1) When you generate using generate.ps1, it generates a meta data file as sdkInfo*.cs
2) After generation you need to build at least once on commandline as
msbuild build.proj /t:Build /p:Scope=AzureStack\UpdateAdmin

File: src/AzureStack/UpdateAdmin/Update.Admin/Microsoft.AzureStack.Management.Update.Admin.csproj (hunk 2 above)
Comment (deathly809) [reply src/AzureStack/UpdateAdmin/Update.Admin/Microsoft.AzureStack.Management.Update.Admin.csproj]: I have no clue,  someone else was working on this.  I will clean it up.

**Comment by deathly809** [comment]
@shahabhijeet I think I have done everything you have asked.

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**